### PR TITLE
Fix:modify uppercase and lowercase issues

### PR DESCRIPTION
**Pull request by FinalT**
Now protoc-gen-go-triple will ignore the case issue and convert to uppercase

### CODE REVIEW

File: cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go [hunk 1]
```diff
@@ -65,7 +65,14 @@ func init() {
...
 		log.Fatal(err)
 	}
-	TplClientInterface, err = template.New("clientInterface").Parse(InterfaceTpl)
+	TplClientInterface, err = template.New("clientInterface").Funcs(template.FuncMap{
+		"upper": func(s string) string {
+			if s == "" {
+				return ""
+			}
+			return strings.ToUpper(s[:1]) + s[1:]
+		},
```
Comment (FoghostCn) [inline cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go:74]: use `util.ToUpper` instead maybe better

File: cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go [hunk 2]
```diff
@@ -76,6 +83,12 @@ func init() {
 			}
 			return strings.ToLower(s[:1]) + s[1:]
 		},
+		"upper": func(s string) string {
+			if s == "" {
+				return ""
+			}
+			return strings.ToUpper(s[:1]) + s[1:]
+		},
```
Comment (FoghostCn) [inline cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go:91]: ditto

File: cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go [hunk 3]
```diff
@@ -99,7 +112,14 @@ func init() {
...
 		log.Fatal(err)
 	}
-	TplHandler, err = template.New("handler").Parse(HandlerTpl)
+	TplHandler, err = template.New("handler").Funcs(template.FuncMap{
+		"upper": func(s string) string {
+			if s == "" {
+				return ""
+			}
+			return strings.ToUpper(s[:1]) + s[1:]
+		},
```
Comment (FoghostCn) [inline cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go:121]: ditto

File: cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go [hunk 4]
```diff
@@ -121,6 +141,12 @@ func init() {
 			}
 			return strings.ToLower(s[:1]) + s[1:]
 		},
+		"upper": func(s string) string {
+			if s == "" {
+				return ""
+			}
+			return strings.ToUpper(s[:1]) + s[1:]
+		},
```
Comment (FoghostCn) [inline cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go:149]: ditto

File: cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go (hunk 1 above)
Comment (FinalT) [reply cmd/protoc-gen-go-triple/gen/generator/tripleTpl.go]: done.

**Pull request merged**